examples/pi-home-router/src/arp/arp_frame.rs

```diff
@@ -0,0 +1,106 @@
+use route_rs_packets::{EthernetFrame, MacAddr};
+use std::net::IpAddr;
+
+pub(crate) enum ArpOp {
+    Request = 1,
+    Reply = 2,
+}
+
+pub(crate) enum ArpHardwareType {
+    Ethernet = 1,
+}
+
+pub(crate) const ARP_ETHER_TYPE: u16 = 0x0806;
+
+// NOTE: Could be implemented in various ways, such as a specialized version of EthernetFrame that's
```

Comment:
I think this is the reasonable way to do this. We should put it in route-rs-packets rather than asking every implementor to write their own ARP frame type.